pkg/asset/installconfig/powervs/session.go

```diff
@@ -76,7 +69,7 @@ func getPISession() (*ibmpisession.IBMPISession, *UserCredentials, error) {
 			},
 		}, &id)
 		if err != nil {
-			return nil, nil, errors.New("Error saving the IBMID variable")
+			return nil, errors.New("Error saving the IBMID variable")
```

Comment:
can you update this to remove the word variable? it's a very programmer-y word :D how about just "Error saving IBM Cloud User ID."?